lib/services/beanstalk/index.js

```diff
@@ -272,10 +305,18 @@ function getSystemInjectedEbExtensions(stackName, ownServiceContext, dependencie
 
 exports.check = function (serviceContext) {
     let errors = [];
+    let params = serviceContext.params;
 
     //TODO - Implement check method
```

Comment:
Ah I see here I've forgotten to implement check for the Beanstalk method. :) Thanks for adding the check for the DNS names stuff! I'll try to improve this with the remaining checks at some point.